### PR TITLE
Add capability of deriving sky temperature from weather file horizontal infrared radiation

### PR DESCRIPTION
**Pull request by chipbarnaby**
## Description

By default, CSE derives sky temperature from total sky cover, dry-bulb temperature, and dew-point temperature using the Berdahl-Martin model.

This PR adds the capability of deriving sky temperature from horizontal infrared radiation available in (some) EPW files.  The capability is preliminary -- missing weatherfile IR data is not detected / diagnosed.

Added an IRHORIZ case to test file wthr01.cse.

Updated documentation.

### CODE REVIEW

File: doc/src/records/top-members.md [hunk 1]
```diff
@@ -532,6 +532,7 @@ DEFAULT,        Default: tSky from weather file if available else Berdahl-Martin
 BERDAHLMARTIN,  Berdahl-Martin (tSky depends on dew point&comma; cloud cover&comma; and hour)
 DRYBULB,   	   tSky = dry-bulb temperature (for testing)
 BLAST,          Blast model (tSky depends on dry-bulb)
+IRHORIZ,        Experimental model that derives tSky for horizonal infrared data from the weather file (available on some EPW files only)
```
Comment (nealkruis) [inline doc/src/records/top-members.md:535]: It's not experimental once it's merged.

File: src/wfpak.cpp [hunk 2]
```diff
@@ -384,6 +384,26 @@ RC WDHR::wd_Unpack(		// single-hour unpack
...
+		pF = fopen(fName, "wt");
+		if (pF)
+			fprintf(pF, "doy,hr,tdb,dni,dhi,ghi\n");
+	}
+	if (pF)
+		fprintf(pF, "%d,%d,%0.1f,%0.1f,%0.1f,%0.1f\n",
+			jDayST, iH, wd_db, wd_DNI, wd_DHI, wd_glrad);
+
+}		// WDHR::wd_WriteCSV
+#endif
```
Comment (nealkruis) [inline src/wfpak.cpp:406]: Not clear what this is or if it's intended to be merged. Looks like temporary test code?

File: src/wfpak.cpp [hunk 3]
```diff
@@ -3064,19 +3088,20 @@ x			printf( "mismatch\n");
...
 		static FILE* pF = NULL;		// file
 		if (!pF)
 		{
 			const char* fName = "tSky.csv";
 			pF = fopen(fName, "wt");
-			fprintf(pF, "yr,mon,day,hr,osc,tsc,tSky_CSE,tSky_IR\n");
+			fprintf(pF, "yr,mon,day,hr,osc,tsc,tSky_CSE (F),tSky_IR (F)\n");
 		}
 		fprintf(pF, "%d,%d,%d,%d,%0.1f,%0.1f,%0.1f,%0.1f\n", yr, m, d, h, osc, tsc, wd_tSky, tSkyIR);
 #endif
```
Comment (nealkruis) [inline src/wfpak.cpp:3107]: Is this still an experiment? I imagine this can all be deleted now?

**Review by nealkruis — CHANGES_REQUESTED** [review]
Code looks fine. Just some clean up of testing/debugging/experimenting needed.

File: src/wfpak.cpp (hunk 3 above)
Comment (chipbarnaby) [reply src/wfpak.cpp]: Updated this code and left it in.  We are not yet done with testing sky models.

File: src/wfpak.cpp (hunk 2 above)
Comment (chipbarnaby) [reply src/wfpak.cpp:406]: Left for possible future use.

**Review by nealkruis — APPROVED** [review]

File: doc/src/records/top-members.md (hunk 1 above)
Comment (chipbarnaby) [reply doc/src/records/top-members.md]: 1. Reworded.
2. I don't agree that there can be no experimental models in merged code.  New stuff has to be merged to allow external users to test.

**Pull request merged**